params/ekf.yaml

```diff
@@ -164,15 +178,15 @@ ekf_filter_node:
         control_config: [true, false, false, false, false, true]
         # TODO : Edit limits for accelarion
 # Places limits on how large the acceleration term will be. Should match your robot's kinematics.
-        acceleration_limits: [1.3, 0.0, 0.0, 0.0, 0.0, 3.4]
+        acceleration_limits: [3.9, 0.0, 0.0, 0.0, 0.0, 3.1]
 # Acceleration and deceleration limits are not always the same for robots.
-        deceleration_limits: [1.3, 0.0, 0.0, 0.0, 0.0, 4.5]
+        deceleration_limits: [3.9, 0.0, 0.0, 0.0, 0.0, 3.1]
 # If your robot cannot instantaneously reach its acceleration limit, the permitted change can be controlled with these
 # gains
-        acceleration_gains: [0.8, 0.0, 0.0, 0.0, 0.0, 0.9]
+        acceleration_gains: [0.5, 0.0, 0.0, 0.0, 0.0, 0.5]
 # If your robot cannot instantaneously reach its deceleration limit, the permitted change can be controlled with these
 # gains
-        deceleration_gains: [1.0, 0.0, 0.0, 0.0, 0.0, 1.0]
+        deceleration_gains: [0.5, 0.0, 0.0, 0.0, 0.0, 0.5]
```

Comment:
Why did you change this values?

Reply:
I calculated the values ​​roughly for the AGV there. Im not sure if its necessary

Reply:
If you want to compare the accuracy of odometry with and without imu data, then maybe we should keep all other parameters the same? So other parameters won't affect the result?